src/components/CurriculumUnitDetails/CurriculumUnitDetails.tsx

```diff
@@ -12,11 +12,19 @@ export type CurriculumUnitDetailsProps = {
   unitTitle?: string;
   threads: Thread[];
   lessons: Lesson[] | null | undefined;
+  previousUnitDescription: string | null;
```

Comment:
So sorry about this, but we should consistently use `prior` rather than `previous` as that is what is in the database.